File: BloodConnector.WebAPI/Controllers/HomeController.cs
using System.Threading.Tasks;
using System.Web.Mvc;

namespace BloodConnector.WebAPI.Controllers
{
    public class HomeController : Controller
    {
        private readonly ApplicationUserManager _userManager;

        public HomeController(ApplicationUserManager userManager)
        {
            _userManager = userManager;
        }

        public ActionResult Index()
        {
            ViewBag.Title = "Home Page";

            return View();
        }

        [AllowAnonymous]
        [HttpPost]
        public async Task<JsonResult> UserAlreadyExistsAsync(string Email)
        {
            var result = await _userManager.FindByEmailAsync(Email);
            return Json(result == null, JsonRequestBehavior.AllowGet);
        }
    }
}

File: BloodConnector.WebAPI/app/controllers/homeController.js
'use strict';
app.controller('homeController', ['dataService', function (dataService) {
    var vm = this;
    vm.usersBloodGroup = [];

    vm.$onInit = function () {
        dataService.getUsersBloodGroup.then(function (result) {
            vm.usersBloodGroup = result.data;
        });
    };

}]);